src/sentry_plugins/base.py

```diff
@@ -18,6 +19,9 @@
     UnsupportedResponseType,
 )
 
+if TYPE_CHECKING:
+    from django.utils.functional import _StrPromise
+
 
 class CorePluginMixin:
     author: str | None = "Sentry Team"
```

Comment:
it's a mixin, so it may be mixed into a class which has them set to `None`

mixins suck, they make everything harder to debug, type, test, and reason about